packages/fxa-shared/metrics/glean/web/index.ts

```diff
@@ -47,6 +47,7 @@ export const eventsMap = {
   loginConfirmation: {
     view: 'login_email_confirmation_view',
     submit: 'login_email_confirmation_submit',
+    success: 'login_email_confirmation_success_view',
```

Comment:
Should this be `login_email_confirmation_success` here, as this isn't a page view? I would expect the `view` event to be triggered on render of the `signin_verified`/`signin_confirmed` pages.

Reply:
The '_view' postfix is sometimes used to distinguish the event from its backend counterpart that does not have the postfix.  I do agree it can be a little confusing though.

Reply:
Oh yeah, that is kinda confusing since 90% of the time we use `view` for page view events. Might have been nice to use something like `client` or `frontend` as differentiator.

Reply:
@ksiegler1 what do you think of suggestions from @vpomerleau ?